packages/mobile/src/components/LineItemRow.tsx

```diff
@@ -0,0 +1,59 @@
+import colors from '@celo/react-components/styles/colors'
```

Comment:
I know we kinda have `src/components` as a generic dumping ground but I would love to add some more structure there

Reply:
Yeah, it's the place where components that probably won't be shared across packages go. Open to other suggestions